testsuite/forge/src/main.rs

```diff
@@ -10,18 +10,19 @@ use diem_sdk::{
 };
 use forge::{forge_main, ForgeConfig, Result, *};
 use itertools::Itertools;
-use rand::SeedableRng;
+use rand::{rngs::OsRng, Rng, SeedableRng};
 use std::time::Duration;
 use tokio::runtime::Runtime;
 
 fn main() -> Result<()> {
     let tests = ForgeConfig {
         public_usage_tests: &[&FundAccount, &TransferCoins],
         admin_tests: &[&GetMetadata],
-        network_tests: &[&RestartValidator, &EmitTransaction],
```

Comment:
:/ We'll need to probably build out configurable testsuites so we can still test the RestartValidator for local stuff